api/database/dbops.js

```diff
@@ -428,13 +429,10 @@ async function saveResponses(
       );
     }
 
-    // if their game ended due to an error, don't mark as completed
-    if (errorEnd) return;
```

Comment:
As i noted in a comment on the PR, not sure we should be blocking people who end on an error.